internal/config/config.go

```diff
@@ -0,0 +1,30 @@
+/*
+SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and cf-service-operator contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"github.com/caarlos0/env/v11"
+)
+
+// Config defines the configuration keys
+type Config struct {
+
+	//Resource cache is enabled or disabled
+	IsResourceCacheEnabled bool `env:"RESOURCE_CACHE_ENABLED" envDefault:"true"`
```

Comment:
By default, we want to have the "old" behaviour without the cache.
Therefore, you should set the default to false.